src/OpenDiffix.Core/Anonymizer.fs

```diff
@@ -63,7 +63,11 @@ let isLowCount (executionContext: ExecutionContext) (aidSets: HashSet<AidHash> s
         [ executionContext.NoiseLayers.BucketSeed; seedFromAidSet aidSet ]
         |> generateNoise anonParams.Salt "suppress" anonParams.Suppression.LayerSD
 
-      let thresholdMean = anonParams.Suppression.LowMeanGap + float anonParams.Suppression.LowThreshold
+      // `LowMeanGap` is the number of standard deviations between `LowThreshold` and desired mean
+      let thresholdMean =
+        anonParams.Suppression.LowMeanGap * anonParams.Suppression.LayerSD
```

Comment:
We have two noise layers, so `TotalSD = LayerSD * sqrt(2)`.

Reply:
Right. I think in this case, I'll also make it a little more explicit in the parameter docs. It's easy to miss the `layer` and expect that `LayerSD` is the `standard deviation` meant here